<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArrowedTextControlWidget</class>
 <widget class="QWidget" name="ArrowedTextControlWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="LineEditWithUnicodeDialog" name="nameLineEdit" native="true"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Text x coord</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="textXCoordLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Text y coord</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="textYCoordLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Arrow x coord</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="arrowXCoordLineEdit"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Arrow y coord</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="arrowYCoordLineEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Pen</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Background color</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="ColorChoiseWidget" name="backgroundColorWidget" native="true"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Arrow style</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="AnyComboBox" name="arrowStyleComboBox" native="true"/>
   </item>
   <item row="5" column="1">
    <widget class="PenChoiceWidget" name="penChoiceWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnyComboBox</class>
   <extends>QWidget</extends>
   <header location="global">commonWidgets/anycombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorChoiseWidget</class>
   <extends>QWidget</extends>
   <header location="global">commonWidgets/colorchoisewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineEditWithUnicodeDialog</class>
   <extends>QWidget</extends>
   <header>commonWidgets/lineeditwithunicodedialog.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PenChoiceWidget</class>
   <extends>QWidget</extends>
   <header>commonWidgets/penchoicewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
